cmd/cdi-cloner/clone-source.go

```diff
@@ -143,8 +144,12 @@ func validateMount() {
 	}
 }
 
-func newTarReader() (io.ReadCloser, error) {
-	cmd := exec.Command("/usr/bin/tar", "Scv", ".")
+func newTarReader(preallocation bool) (io.ReadCloser, error) {
+	args := "cv"
+	if !preallocation {
```

Comment:
a very short comment that preallocation does not play well with sparse tar files

Reply:
Done